calabash/Classes/FranklyServer/Routes/LPScreenshotRoute.m

```diff
@@ -32,7 +32,14 @@ - (BOOL) supportsMethod:(NSString *) method atPath:(NSString *) path {
 
 - (NSData *) takeScreenshot {
 
-  CGSize imageSize = [[UIScreen mainScreen] bounds].size;
+
+  // Available on iOS >= 7
+  SEL selector = @selector(drawViewHierarchyInRect:afterScreenUpdates:);
+  UIView *view = [[UIView alloc] initWithFrame:CGRectZero];
+  BOOL drawViewHierarchy = [view respondsToSelector:selector];
```

Comment:
I thought about asking the `[LPTouchUtils applicationWindows][0]` window if it responded to that selector, but decided this was a reasonable way to test for availability.  I am, on the other hand, suffering from vertigo and feeling super muddled.  @sapieneptus What say you?